Bindings/Java/tests/TestBasics.java

```diff
@@ -3,6 +3,7 @@
 class TestBasics {
     public static void testBasics() {
         Model m = new Model();
+        m.finalizeFromProperties();
```

Comment:
It is necessary because a few lines down we want to use `getComponentList()` (invoked by `m.getFrameList()`). We are asking a `Model` to list its subcomponents without giving it an opportunity to identify its subcomponents amongst it properties (all those Sets) and other data members. Again we were calling `finalizeFromProperties()` in the constructor before (thus hiding the fact) but the consensus was that it was not consistent with copying a model.